### PR TITLE
TemporaryJobs: Change the test report output dir

### PR DESCRIPTION
**Pull request by davidxia**
Instead of .helios-reports/ use target/helios-reports
because people complained. :(

### CODE REVIEW

**Comment by davidxia** [comment]
@rculbertson

**Comment by davidxia** [comment]
@gilles

File: helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java [hunk 1]
```diff
@@ -87,7 +87,7 @@
   private static final Prober DEFAULT_PROBER = new DefaultProber();
   private static final String DEFAULT_LOCAL_HOST_FILTER = ".+";
   private static final String DEFAULT_PREFIX_DIRECTORY = "/tmp/helios-temp-jobs";
-  private static final String DEFAULT_TEST_REPORT_DIRECTORY = "./helios-reports/test";
+  private static final String DEFAULT_TEST_REPORT_DIRECTORY = "target/helios-reports/test";
```
Comment (gilles) [inline helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java]: This assumes you're running with maven or any build system that relies on the maven directory structure

Comment (davidxia) [inline helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java]: Perhaps it's better to just have users remember to configure it to output to target or add to .gitignore?

Comment (gilles) [inline helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java]: Maybe this could be null which means disabled by default?

**Comment by rohansingh** [comment]
:+1:

File: helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java (hunk 1 above)
Comment (gimaker) [inline helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java]: if you don't use maven, and consequently don't have the target directory, then the only thing that will happen is that we create the target directory and put stuff in there. is that really so bad?

**Comment by gimaker** [comment]
:+1:

**Pull request merged**